ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -29,6 +30,10 @@
  * class doesn't implement {@link ReactApplication}.
  */
 public class ReactActivityDelegate {
+
+  private final int REQUEST_OVERLAY_CODE = 1111;
```

Comment:
Anyway, I don't think this is important enough to bikeshed on. Please do the changes @SandroMachado mentioned and I'll merge this.